cypress/elements/catalog/product-selectors.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable sort-keys */
 export const PRODUCTS_SELECTORS = {
+  productsList: "[data-test-id][data-test='id']",
```

Comment:
```suggestion
  productsList: "[data-test='id']",
```

`[data-test-id]` is redundant here IMO

Reply:
[data-test-id] is necessary. Before all products are loaded there is an empty row on products table with [data-test='id']. Without [data-test-id] once in a while test will crush because it will try to click on that empty row.